lib/src/rpc/dial.dart

```diff
@@ -113,6 +136,30 @@ Future<ClientChannelBase> dial(String address, DialOptions? options, String Func
   return _dialWebRtc(address, opts, sessionCallback);
 }
 
+Future<String> searchMdns(String address) async {
+  // We need to replace all periods with dashes, because this is how viam instances are broadcast locally.
```

Comment:
(q) Is this always the case? I often see both the `.` and `-` versions when querying mDNS.

Reply:
I have seen both as well, but with the bonsoir package I am using it was only finding the `-` version.